### PR TITLE
Sanitize input closes #68

### LINKED ISSUE #68: input sanitization

### PR DESCRIPTION
**Pull request by kerrd89**
The spacing looks off here on github, but input sanitization appears to be working.

My approach:
I found a blog on the subject and sent the reg-ex function from that to alex to ask for his opinion.  He suggested we look into sanitize-html, but provided a functional alternative as well.

I tried a npm module called 'sanitize-html', but there were peer dependencies which were causing issues. 

The function I found on the aforementioned blog worked for rendering traditional html, but doesn't work for react (replacing symbols with their text equivalent).

Long story short, I am using the function alex provided and it appears to do the job.  We still need to validate email though seperately on both the newAdmin side and the newSpike side.

### CODE REVIEW

**Comment by Jeff-Duke** [comment]
This all looks good.  I pulled down the branch and tried to inject some scripts etc, I wasn't able to.
